dotlottie/src/main/java/com/lottiefiles/dotlottie/core/widget/DotLottieAnimation.kt

```diff
@@ -205,7 +220,8 @@ class DotLottieAnimation @JvmOverloads constructor(
                         useFrameInterpolation = config.useFrameInterpolator,
                         backgroundColor = Color.TRANSPARENT.toUInt(),
                         segments = listOf(),
-                        marker = config.marker
+                        marker = config.marker,
+                        layout = Layout(Fit.FILL, listOf(0.5f, 0.5f)),
```

Comment:
you can use `createDefaultLayout` instead from the dotlottie rs core

```suggestion
                        layout = createDefaultLayout(),
```